src/components/docs/versions/date-time.tsx

```diff
@@ -10,8 +10,9 @@ const options: Intl.DateTimeFormatOptions = {
   day: 'numeric',
   hour: 'numeric',
   minute: 'numeric',
-  second: 'numeric',
   timeZoneName: 'short',
+  timeZone: 'UTC',
```

Comment:
Could you explain why this changed to UTC? I don't think it's useful if everyone has to manually calculate their timezone to UTC; not very user friendly.

Reply:
It shortened the string a bit, but I realized I should just move it to a tooltip instead so that's what I'm doing now.